packages/mui-material/src/CardContent/CardContent.js

```diff
@@ -22,9 +22,9 @@ const CardContentRoot = styled('div', {
   overridesResolver: (props, styles) => styles.root,
 })(() => {
   return {
-    padding: 16,
+    padding: styled.theme.spacing(2),
```

Comment:
`TypeError: Cannot read property 'spacing' of undefined`

Reply:
It should be

```js
})(({ theme }) => {
  return {
    padding: theme.spacing(2),
```